lib/qiita2evernote/cli.rb

```diff
@@ -0,0 +1,114 @@
+require 'thor'
+require 'dotenv'
+
+require_relative './client'
+require_relative './error'
+
+class Qiita2Evernote::Cli < Thor
+
+  desc "init", "initialize"
+  def init
```

Comment:
__[rubocop]__
* Metrics/CyclomaticComplexity: Cyclomatic complexity for init is too high. [16/10]
* Metrics/PerceivedComplexity: Perceived complexity for init is too high. [16/7]

<!-- Comment by [SideCI](https://sideci.com) -->